vscode/src/completions/get-inline-completions-tests/multiline-truncation.test.ts

```diff
@@ -681,6 +681,73 @@ cases.forEach(isTreeSitterEnabled => {
                       }"
                 `)
                 })
+
+                it('handles missing brackets gracefully to truncate the completion correctly', async () => {
+                    const requestParams = params('console.log(1); const █', [completion``], {
+                        onNetworkRequest(_params, onPartialResponse) {
+                            onPartialResponse?.(completion`
+                                ├MyCoolObject = {
+                                constructor() {
+                            ┤`)
+                            onPartialResponse?.(completion`
+                                ├MyCoolObject = {
+                                constructor() {
+                                    console.log(1)
+
+                                    if (false
+                            `)
```

Comment:
This does not look like a realistic partial response. I.e if the line `if (false` is only partially ready, where is there a newline afterwards?

Should this be:

```suggestion
                            onPartialResponse?.(completion`
                                ├MyCoolObject = {
                                constructor() {
                                    console.log(1)

                                    if (false`)
```

instead?

Reply:
Yep, good catch!  There should be no newline at the end.